.gitignore

```diff
@@ -17,3 +17,4 @@ hello_world/host/optee_example_hello_world
 random/host/optee_example_random
 aes/host/optee_example_aes
 hotp/host/optee_example_hotp
+secure_storage/host/optee_example_secure_storag
```

Comment:
Is an `e` missing at the end?

Reply:
Good catch. i don't know how I corrupted that!
Thanks.